app/src/main/java/com/example/sharewhatyoucanproject/DrawerActivity.kt

```diff
@@ -1,11 +1,128 @@
 package com.example.sharewhatyoucanproject
 
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.ProgressDialog
+import android.content.Intent
+import android.os.Build
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.view.Menu
+import android.view.Window
+import android.widget.TextView
+import androidx.drawerlayout.widget.DrawerLayout
+import androidx.navigation.findNavController
+import androidx.navigation.ui.AppBarConfiguration
+import androidx.navigation.ui.navigateUp
+import androidx.navigation.ui.setupActionBarWithNavController
+import androidx.navigation.ui.setupWithNavController
+import com.example.sharewhatyoucanproject.databinding.ActivityDrawerBinding
+import com.google.android.material.navigation.NavigationView
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.messaging.FirebaseMessaging
 
 class DrawerActivity : AppCompatActivity() {
-    override fun onCreate(savedInstanceState: Bundle?) {
+    private lateinit var appBarConfiguration: AppBarConfiguration
+    private lateinit var binding: ActivityDrawerBinding
+    lateinit var pd: ProgressDialog
+    lateinit var usernameh: TextView
+
+    protected override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_drawer)
+
+        pd = ProgressDialog(this)
+        pd.setTitle("Logging Out")
+        pd.setCancelable(false)
+
+
+
+        binding = ActivityDrawerBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        supportActionBar?.hide()
+        // requestWindowFeature(Window.FEATURE_NO_TITLE);
```

Comment:
Don't merge dead code to your main branch.  If this is something you need for later, instead I would create a ticket to track it.